_sass/_va.scss

```diff
@@ -332,7 +332,7 @@ a:hover {text-decoration: underline; background: rgba(0,0,0,.1);}
 }
 
 html.no-touch .banner {
-  background: url(../img/design/banner.jpg) 50% 60% no-repeat;
+//  background: url(../img/design/banner.jpg) 50% 60% no-repeat;
```

Comment:
This actually just looks like broken CSS in general.  @akainic @emilyville thoughts?